compiler/lsp-tests/src/Main.hs

```diff
@@ -398,6 +398,28 @@ requestTests run _runScenarios = testGroup "requests"
           -- answerFromTest
           locs <- getDefinitions main' (Position 2 8)
           liftIO $ locs @?= [Location (test ^. uri) (Range (Position 1 0) (Position 1 14))]
+
+          -- introduce syntax error
+          changeDoc main' [TextDocumentContentChangeEvent (Just (Range (Position 6 6) (Position 6 6))) Nothing "+"]
```

Comment:
It would be great to insert a newline in addition to this to make sure position remapping works correctly.

Reply:
ah good idea, I'll do that.